pkg/cmd/artefactor.go

```diff
@@ -29,6 +30,14 @@ const (
 	// FlagImageVars is used to specify a whitelist of variable names to enable
 	// "export"
 	FlagImageVars = "image-vars"
+	// FlagDockerUserName overrides docker registry configuration
+	FlagDockerUserName = "docker-username"
```

Comment:
README should be updated for the new flags with a little note on use-cases.

Reply:
will do

Reply:
Pushed up - please check @KashifSaadat